src/Standards/Squiz/Docs/WhiteSpace/SuperfluousWhitespaceStandard.xml

```diff
@@ -0,0 +1,99 @@
+<documentation title="Superfluous Whitespace">
+    <standard>
+    <![CDATA[
+    There should be no superfluous whitespace at the start of a file.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No whitespace preceding first content in file.">
+        <![CDATA[<?php
+echo 'opening PHP tag at start of file';
+        ]]>
+        </code>
+        <code title="Invalid: Whitespace used before content in file.">
+        <![CDATA[
+<em>        </em>
+<?php
+echo 'whitespace before opening PHP tag';
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There should be no trailing whitespace at the end of lines.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: No whitespace found at end of line.">
+        <![CDATA[
+echo 'semicolon followed by new linechar';
+        ]]>
+        </code>
+        <code title="Invalid: Whitespace found at end of line.">
+        <![CDATA[
+echo 'trailing spaces after semicolon';<em>   </em>
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There should be no more than one consecutive blank newline in functions.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Functions do not contain multiple empty lines in a row.">
+        <![CDATA[
+function myFunction()
+{
+    echo 'code here';
+
+    echo 'code here';
+}
+        ]]>
+        </code>
+        <code title="Invalid: Functions contain multiple empty lines in a row.">
+        <![CDATA[
+function myFunction()
+{
+    echo 'code here';
+    <em>
+
+    </em>echo 'code here';
+}
+        ]]>
+        </code>
+    </code_comparison>
+    <standard>
+    <![CDATA[
+    There should be no superfluous whitespace at the end of a file.
+    A single final newline is allowed. A closing PHP tag is optional and ignored.
```

Comment:
This description is incorrect and neither of the code samples would trigger the sniff.

As per my previous review:

> It's basically about whitespace found after a PHP close tag (without inline HTML contents after).

So that rule is not about whitespace between the last code and the close tag, but about whitespace _after a close tag when the close tag is not followed by inline HTML.

Does that help ?

Reply:
Apologies. I've clearly gotten confused somewhere. I'll try to be more careful in future. I've rewritten this in a584db264. Thanks for your patience.

Reply:
Happens to the best of us. Looks much better now.